src/lib/apiTypesV2/Attribute.cpp

```diff
@@ -52,7 +52,8 @@ std::string Attribute::render(ConnectionInfo* ciP, RequestType requestType, bool
     else
     {
       out = "{";
-      out += pcontextAttribute->toJson(true, renderMode, requestType);  // param 1 'true' as it is the last and only element
+      // FIXME PR: renderMode + attrsFormat ... a bit confusing. Should we join these two somehow? 
```

Comment:
Yes, it makes sense.
The render() method of ContextAttribute is pretty complex and we need to look at this carefully